nav2_lifecycle_manager/README.md

```diff
@@ -0,0 +1,23 @@
+#### Background on lifecycle enabled nodes
+The lifecycle enabled nodes in the navigation stack (e.g. map_server, planner_server, controller_server) have lifecycle functions namely on_configure(), on_activate(), on_deactivate(), on_cleanup(), on_shutdown(), and on_error(). Managed lifecycle nodes allow roslaunch to ensure that all components have been instantiated correctly before it allows any component to begin executing its behaviour. It will also allow nodes to be restarted or replaced on-line. More details about managed nodes can be found on [ROS2 Design website](https://design.ros2.org/articles/node_lifecycle.html).

```

Comment:
- the methods could be code, ei `on_configure()` not on_configure() 
- ros2launch
- Careful when you say "components" lifecycle nodes and component nodes aren't related

Reply:
Done. Renamed components to nodes.